scripts/analysis-kevm.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+
+export PYTEST_PARALLEL=8 
+
+BUG_REPORT_DIR=${BUG_REPORT_DIR:-$($SCRIPT_DIR/performance-tests-kevm.sh --bug-report)}
+
+echo "Bug reports saved at $BUG_REPORT_DIR"
+
+nix_shell() {
+  GC_DONT_GC=1 nix develop $SCRIPT_DIR/..#cabal --extra-experimental-features 'nix-command flakes' --command bash -c "$1"
+}
+
+nix_shell "cabal build kore-rpc-booster"
+
+export SERVER=$(nix_shell "cabal exec which kore-rpc-booster")
+
+nix_shell "cabal build kore-rpc-client"
+
+export CLIENT=$(nix_shell "cabal exec which kore-rpc-client")
+
+export LLVM_LIB="$BUG_REPORT_DIR/interpreter.dylib"
+
+export LOG_DIR="$BUG_REPORT_DIR"
+
+for tar in $(find $BUG_REPORT_DIR -name \*tar ); 
+do 
+  echo "######## $tar ########"; 
+  nix_shell "$SCRIPT_DIR/run-with-tarball.sh $tar -l Aborts --print-stats 2>&1 | tee $BUG_REPORT_DIR/$(basename $tar).out"; 
+done
+
+cd $BUG_REPORT_DIR
```

Comment:
I would suggest using a separate directory (`$LOG_DIR`?) for the outputs `*.log` and `*.out`, creating one unless an existing one was given by the user.
If we write the output to `$BUG_REPORT_DIR` it will be overwritten by a new run and clean-up is easier if you want to keep the tarballs.